src/uct/self/self_iface.c

```diff
@@ -19,14 +19,18 @@ static ucs_config_field_t uct_self_iface_config_table[] = {
 
 static ucs_status_t uct_self_iface_query(uct_iface_h iface, uct_iface_attr_t *attr)
 {
+    uct_self_iface_t *self_iface = ucs_derived_of(iface, uct_self_iface_t);
+
     ucs_trace_func("iface=%p", iface);
     memset(attr, 0, sizeof(*attr));
 
     attr->iface_addr_len         = sizeof(uct_self_iface_addr_t);
     attr->device_addr_len        = 0;
     attr->ep_addr_len            = 0; /* No UCT_IFACE_FLAG_CONNECT_TO_EP supported */
-    attr->cap.flags              = UCT_IFACE_FLAG_CONNECT_TO_IFACE |
-                                   UCT_IFACE_FLAG_AM_SHORT         |
+    attr->cap.flags              = UCT_IFACE_FLAG_CONNECT_TO_IFACE    |
+                                   UCT_IFACE_FLAG_AM_SHORT            |
+                                   UCT_IFACE_FLAG_AM_BCOPY            |
+                                   UCT_IFACE_FLAG_ERRHANDLE_BCOPY_LEN |
```

Comment:
need to remove this flag because we cannot know the size of the message before copying